solutions/standard/variables.tf

```diff
@@ -73,6 +196,16 @@ variable "iam_engine_name" {
   default     = "base-sm-iam-engine"
 }
 
+variable "endpoint_type" {
```

Comment:
no need to expose this - this can be determined based on the value used for the `allowed_network` variable.